content/docs/notebooks/setup.md

```diff
@@ -0,0 +1,251 @@
+  interface. If you requested a new workspace, the dashboard should be empty
+  of notebooks:
+
+    <img src="/docs/images/jupyter-dashboard.png" 
+      alt="Jupyter dashboard with no notebooks"
+      class="mt-3 mb-3 border border-info rounded">
+
+1. Click **Upload** to upload an existing notebook, or click **New** to
+  create an empty notebook. You can read about using notebooks in the
+  [Jupyter documentation](https://jupyter-notebook.readthedocs.io/en/latest/notebook.html#notebook-user-interface).
+
+## Experiment with your notebook
+
+The default notebook image includes all the plugins that you need to train a 
+Tensorflow model with Jupyter, including
+[Tensorboard](https://www.tensorflow.org/get_started/summaries_and_tensorboard) 
+for rich visualizations and insights into your model.
+
+To test your Jupyter installation, you can run a basic 'hello world' program
+(adapted from
+[mnist_softmax.py](https://github.com/tensorflow/tensorflow/blob/r1.4/tensorflow/examples/tutorials/mnist/mnist_softmax.py)) as follows:
+
+1. Use the Jupyter dashboard to create a new **Python 3** notebook.
+
+1. Copy the following code and paste it into a code block in your notebook:
+
+    ```
+    from tensorflow.examples.tutorials.mnist import input_data
+    mnist = input_data.read_data_sets("MNIST_data/", one_hot=True)
+
+    import tensorflow as tf
+
+    x = tf.placeholder(tf.float32, [None, 784])
+
+    W = tf.Variable(tf.zeros([784, 10]))
+    b = tf.Variable(tf.zeros([10]))
+
+    y = tf.nn.softmax(tf.matmul(x, W) + b)
+
+    y_ = tf.placeholder(tf.float32, [None, 10])
+    cross_entropy = tf.reduce_mean(-tf.reduce_sum(y_ * tf.log(y), reduction_indices=[1]))
+
+    train_step = tf.train.GradientDescentOptimizer(0.05).minimize(cross_entropy)
+
+    sess = tf.InteractiveSession()
+    tf.global_variables_initializer().run()
+
+    for _ in range(1000):
+      batch_xs, batch_ys = mnist.train.next_batch(100)
+      sess.run(train_step, feed_dict={x: batch_xs, y_: batch_ys})
+
+    correct_prediction = tf.equal(tf.argmax(y,1), tf.argmax(y_,1))
+    accuracy = tf.reduce_mean(tf.cast(correct_prediction, tf.float32))
+    print("Accuracy: ", sess.run(accuracy, feed_dict={x: mnist.test.images, y_: mnist.test.labels}))
+    ```
+
+1. Run the code. You should see a number of `WARNING` messages from TensorFlow,
+  followed by a line showing a training accuracy something like this:
+
+    ```
+    Accuracy:  0.9012
+    ```
+
+Please note that when running on most cloud providers, the public IP address is 
+exposed to the internet and is an unsecured endpoint by default.
+
+## Next steps
+
+* Explore [Kubeflow Fairing](/docs/fairing/) for a complete solution to 
+  building, training, and deploying an ML model from a notebook.
+* Learn the advanced features available from a Kubeflow notebook, such as
+  [submitting Kubernetes resources](/docs/notebooks/submit-kubernetes/) or
+  [building Docker images](/docs/notebooks/docker-image/). 
```

Comment:
This link is broken in the preview. I think this may be /docs/notebooks/submit-docker-image/

Reply:
Thanks! I changed the file name but forgot to change the link.